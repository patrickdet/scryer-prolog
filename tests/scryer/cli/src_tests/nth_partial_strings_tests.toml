args = ["-f", "--no-add-history", "src/tests/nth_partial_strings.pl", "-f", "-g", "use_module(library(nth_partial_strings_tests)),nth_partial_strings_tests:main_quiet(nth_partial_strings_tests)"]
