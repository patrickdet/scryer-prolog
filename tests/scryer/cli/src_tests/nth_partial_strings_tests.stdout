[A,A]65
All tests passed